README.md

```diff
@@ -122,6 +122,8 @@ Additional option to control `rpmlint` behaviour is the addition of `rpmlintrc`
 which uses old syntax for compatibility with old `rpmlint` releases, yet
 it can be normal `toml` file if you wish:
 
-`setBadness('check', 0)`
-
-`addFilter('test-i-ignore')`
```

Comment:
Please preserve the monospace for these lines.

Reply:
It is preserved via 4 space indentation (which is IMO more readble). See the preview here https://github.com/rpm-software-management/rpmlint/blob/a8660e981299588417c3d27afd84ec6923e19c86/README.md

Reply:
All right. So please rebase the pull request, we have conflicts now.